### PR TITLE
Setup 1.5.x builds

### PR DESCRIPTION
**Pull request by tleyden**
See https://github.com/couchbase/sync_gateway/issues/2918

@adamcfraser should this be 1.5.0 or 1.5.1?  I take it the 1.5.0 binary was built directly from master?  

The 1.5.0.xml was copied unmodified from default.xml

### CODE REVIEW

**Comment by adamcfraser** [comment]
1.5.0 is correct.  

You need to update the sync_gateway entry in the 1.5.0 manifest to point to the release/1.5.0 branch (which also still needs to get created).

**Comment by tleyden** [comment]
Yikes, thanks for catching that.  Fixed.

**Comment by tleyden** [comment]
I pushed a new commit to produce 1.5.1 binaries for master branch builds

File: manifest/default.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
   <!-- Build Scripts (required on CI servers) -->
   <project name="build" path="cbbuild" remote="couchbase">
-    <annotation name="VERSION" value="1.5.0"     keep="true"/>
+    <annotation name="VERSION" value="1.5.1"     keep="true"/>
```
Comment (tleyden) [inline manifest/default.xml:26]: @ceejatec is this the only thing needed to have the master branch SG builds named 1.5.1-x?  I didn't see any version info under the product-config.json for default.xml

Comment (ceejatec) [reply manifest/default.xml:26]: Yes, that should be it for master, I believe.

Comment (tleyden) [reply manifest/default.xml:26]: @ceejatec We noticed that the latest master build was 1.5.1-587 -- is there any way to reset the counter for master builds?

**Pull request merged**